src/wallet_setup/CreateNewWallet.tsx

```diff
@@ -38,6 +38,13 @@ const CreateNewWallet = function CreateNewWallet() {
   const [confirmPhrase, setConfirmPhrase]: [string, (confirmPhrase: string) => void] = React.useState<string>('');
   const handleConfirmPhrase = (event: React.ChangeEvent<HTMLInputElement>) => {
     setConfirmPhrase(event.target.value);
+
+    const elem = document.getElementById('create-phrase-phrase-input');
+    if (elem && event.target.value !== '') {
+      elem.style.color = 'white';
+    } else if (elem) {
+      elem.style.color = '#55595c'; // bs-body-color
```

Comment:
Change value to `var(--bs-body-color)` or find a Bootstrap class name with the same effect